tests/unit/commands/generate-test.js

```diff
@@ -47,9 +49,20 @@ describe('generate command', function() {
 
   it('runs GenerateFromBlueprint but with null nodeModulesPath', function() {
     command.project.hasDependencies = function() { return false; };
+    let originalYarnLockPath = path.join(command.project.root, 'yarn.lock');
+    let dummyYarnLockPath = path.join(command.project.root, 'foo.bar');
+    fs.renameSync(originalYarnLockPath, dummyYarnLockPath);
 
     return expect(command.validateAndRun(['controller', 'foo'])).to.be.rejected.then(reason => {
       expect(reason.message).to.eql('node_modules appears empty, you may need to run `npm install`');
+    }).then(() => fs.renameSync(dummyYarnLockPath, originalYarnLockPath));
```

Comment:
since they are reassigned in every `beforeEach()` block there is IMHO no need to reset them in `afterEach()`